zulipterminal/model.py

```diff
@@ -516,6 +516,45 @@ def send_stream_message(self, stream: str, topic: str, content: str) -> bool:
             notify_if_message_sent_outside_narrow(composition, self.controller)
         return message_was_sent
 
+    def prevent_compose_box_in_stream(self, stream_identifier: Any) -> bool:
```

Comment:
If you're unsure of a type, you can check similar methods that use it. This is an `int`, and we normally use `stream_id`.